.gitignore

```diff
@@ -40,3 +40,4 @@ coverage
 libpeerconnection.log
 /config/application.yml
 node_modules
+/vendor/bundle
```

Comment:
I guess this is not going to cause problems - but does it just to apply to how your dev environment is set up? For me at least I  have a `/vendor` dir but not a `bundle` subdir in the repo. If it's just for you maybe this can go in `.git/info/exclude`, or a separate .gitignore in /vendor to solve your specific git issue.